### ISSUE TITLE
NimBLERemoteValueAttribute.cpp needs a static LOG_TAG

### ISSUE THREAD

**Issue by gkoh** [h2zero/NimBLE-Arduino#766]
I use `LOG_TAG` all over the place to drive `ESP_LOGx()`.
With 1.4.3 I get a `LOG_TAG` collision:
```
multiple definition of `LOG_TAG';
...
NimBLE-Arduino/src/NimBLERemoteValueAttribute.cpp:17: first defined here
```

I think this is because it is declared like this:
```
const char* LOG_TAG = "NimBLERemoteValueAttribute";
```

when it needs file scope similar to other classes, eg.:
```
static const char* LOG_TAG = "NimBLEServer";
```

**Comment by h2zero** [comment]
Thanks, will fix this asap

**Comment by h2zero** [comment]
Fixed in [ca26070](https://github.com/h2zero/NimBLE-Arduino/commit/ca2607070fd9d8dbc39bd025acde2a4eeb25fa3c)
